### PR TITLE
Makes `scripts` Windows-compatible.

### PR DESCRIPTION
**Pull request by wtgtybhertgeghgtwtg**
Bumps `jest`, adds [`mkdirp`](https://github.com/substack/node-mkdirp) and [`rimraf`](https://github.com/isaacs/rimraf).

### CODE REVIEW

File: cli/package.json [hunk 1]
```diff
@@ -46,7 +46,6 @@
     "eslint-plugin-flow-vars": "^0.4.0",
     "flow-bin": "^0.32",
     "jest-cli": "^17.0.3",
-    "mkdirp": "^0.5.1",
```
Comment (ryyppy) [inline cli/package.json]: Why is it removed?

Comment (wtgtybhertgeghgtwtg) [inline cli/package.json]: I noticed it was already in `dependencies`.

Comment (ryyppy) [inline cli/package.json]: Ah, true... I don't like the diffing preview sometimes :-)

**Comment by ryyppy** [comment]
Okay I just tested it on OSX as well... looking good, thanks! 👍  ... I will need to check back with other flow-typed maintainers if we want to go the `src for testing` way.. I would actually prefer it

**Pull request merged**